tests/console/force_scheduled_tasks.pm

```diff
@@ -23,16 +23,18 @@ use warnings;
 use testapi;
 use utils 'assert_screen_with_soft_timeout';
 use version_utils 'is_jeos';
+use Utils::Architectures qw(is_aarch64);
 
 sub settle_load {
     my $loop = 'read load dummy < /proc/loadavg  ; top -n1 -b| head -n30 ; test "${load/./}" -lt $limit && break ; sleep 5';
     script_run "limit=10; for c in `seq 1 200`; do $loop; done; echo TOP-DONE > /dev/$serialdev", 0;
     my $before = time;
-    wait_serial('TOP-DONE', 1005) || die 'load not settled';
+    wait_serial('TOP-DONE', is_aarch64() ? 1500 : 1005) || die 'load not settled';
     # JeOS is different to SLE general as it extends the appliance's disk on first boot,
     # so the balance is a different challenge to SLE. Elapsed time is not necessary a key
     # measure here, responsiveness of the system is.
     record_soft_failure 'bsc#1063638' if (time - $before) > (is_jeos() ? 180 : 70) && get_var('SOFTFAIL_BSC1063638');
+    record_soft_failure 'bsc#1178761' if ((time - $before) > 1005);
```

Comment:
same as one line before this should be limited to the "system_performance" scenario to prevent a random soft-fail in random scenarios. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5024 for more details.

Reply:
This issue happen in aarch64 in scenarios that take longer to run in SLE-15-SP3. It might be beneficial in other scenarios in the same architecture for other products, so I believe it is better not to narrow yet. For us is fine, we only detected it in two scenarios in our group. I don't see the advantage.

Reply:
@okurz If we see nothing happening in the bug, we might consider hiding it, but as of now it fails randomly and soft failure will help us to see how often it happens.

Reply:
I was just informing you what happened in the past and why we have the "system_performance" scenario. It will basically be your own choice of pain regarding the different usual test review teams, doesn't bug me :D

Reply:
Yeah, I remember that story, so for now our hope is that we will revert this change completely in a month. If not, we will invest time on some scalable solution, like adjust those timeouts as per `TIMEOUT_SCALE`, etc.
Thanks for your input!